sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-joins.sql

```diff
@@ -51,6 +51,18 @@ create temporary view t3 as select * from values
   ("val3b", 8S, null, 19L, float(17), 25D, 26E2, timestamp '2015-05-04 01:02:00.000', date '2015-05-04')
   as t3(t3a, t3b, t3c, t3d, t3e, t3f, t3g, t3h, t3i);
 
+create temporary view s1 as select * from values
+    (1), (3), (5), (7), (9)
+  as s1(id);
+
+create temporary view s2 as select * from values
+    (1), (3), (4), (6), (9)
+  as s2(id);
+
+create temporary view s3 as select * from values
+    (3), (4), (6), (9)
+  as s3(id);
+
```

Comment:
Plz drop view in the end?

Reply:
> Plz drop view in the end?

About t1, t2, t3, seems in other test file, don't drop table.